src/useradd.c

```diff
@@ -1377,6 +1377,7 @@ static void process_flags (int argc, char **argv)
 				if (!streq(optarg, "")
 				     && '*'  != optarg[0]
 				     && !streq(optarg, "/sbin/nologin")
+				     && !streq(optarg, "/usr/sbin/nologin")
```

Comment:
Don't such systems have a symlink /sbin => /usr/sbin?

I'm okay with the change, but is it really necessary?

Reply:
Although yes, one may specify it as /usr/... in the command line, in which case we need to support it.